Tests/Admin/Set-RsDatabase.Tests.ps1

```diff
@@ -26,7 +26,12 @@ Describe "Set-RsDatabase" {
         $databaseServerName = 'localhost'
         $databaseName = 'ReportServer' + [System.DateTime]::Now.Ticks
         $credentialType = 'ServiceAccount'
-        Set-RsDatabase -DatabaseServerName $databaseServerName -DatabaseName $databaseName -DatabaseCredentialType $credentialType -Confirm:$false -Verbose -ReportServerInstance PBIRS -ReportServerVersion SQLServervNext
+
+        It 'Should throw exception when asking for confirm in noninteractive mode' {
```

Comment:
Can this pattern be used for the rest of the tests?

Reply:
it could, but existing tests shouldn't be modified as they represent behavior that it might be being used already

so the test case with Confirm:$false should stay and pass and add a new one for the force behaviors you are adding